lib/src/html_input_stream.dart

```diff
@@ -307,15 +334,15 @@ bool _hasUtf8Bom(List<int> bytes, [int offset = 0, int length]) {
 /// Decodes the [bytes] with the provided [encoding] and returns an iterable for
```

Comment:
nit: Comment should reflect new code return type.

Reply:
Done.